projects/plugins/jetpack-mu-wpcom/src/features/coming-soon/test/class-coming-soon-test.php

```diff
@@ -0,0 +1,266 @@
+<?php
+/**
+ * Coming Soon Tests File
+ *
+ * @package full-site-editing-plugin
+ */
+
+namespace A8C\FSE\Coming_soon;
+
+require_once __DIR__ . '/../coming-soon.php';
+
+/**
+ * Class Coming_Soon_Test
+ */
+class Coming_Soon_Test extends \WorDBless\BaseTestCase {
+	/**
+	 * Preview links used to test bypassing Coming Soon using the URL with share code provided as GET parameter.
+	 *
+	 * @var \string[][]
+	 */
+	private static $preview_links = array(
+		array(
+			'code'       => 'sharing-code',
+			'created_at' => '2022-11-23',
+		),
+	);
+
+	/**
+	 * Post-test suite actions.
+	 *
+	 * @afterClass
+	 */
+	public static function tear_down_after_class() {
+		self::delete_coming_soon_site_options();
+		parent::tearDownAfterClass();
```

Comment:
This seems wrong. If we expect there to be a parent `tear_down_after_class()` we should call that, otherwise we should call nothing.

Reply:
dab455599229d14649a03027a2f98d6801aa3643

Though one test isn't passing at the moment, haven't looked into it or tried seeing how the test ran from Calypso.